src/components/SearchStatus/Searchstatus.tsx

```diff
@@ -0,0 +1,40 @@
+import { Box, Typography } from "@mui/material";
+import React from "react";
+import { useStyles } from "./searchStatusStyles";
+
+type status = "RecentScreen" | "ResultsScreen";
+
+type SearchStatusPropTypes = {
+  status: status;
+  onhandleClearSearch?: () => void;
+};
+
+export default function SearchStatus({ status, onhandleClearSearch }: SearchStatusPropTypes) {
```

Comment:
This component is incorrect. It is part of a component called `RecentSearch`. That component has 3 items
1. RecentSearchHeader
2. List of SearchResultItem
3. ViewMoreButton

Make necessary changes and delete this component (and its corresponding story) from here.